### PR TITLE
Database updates to populate rooms.room_version

### PR DESCRIPTION
**Pull request by richvdh**
We're going to need this so that we can figure out how to handle redactions when fetching events from the database.

### CODE REVIEW

File: synapse/storage/data_stores/main/schema/delta/57/rooms_version_column_2.sql.postgres [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/* Copyright 2020 The Matrix.org Foundation C.I.C
```
Comment (anoadragon453) [inline synapse/storage/data_stores/main/schema/delta/57/rooms_version_column_2.sql.postgres]:
```suggestion
/* Copyright 2020 The Matrix.org Foundation C.I.C.
```

File: synapse/storage/data_stores/main/schema/delta/57/rooms_version_column_2.sql.sqlite [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/* Copyright 2020 The Matrix.org Foundation C.I.C
```
Comment (anoadragon453) [inline synapse/storage/data_stores/main/schema/delta/57/rooms_version_column_2.sql.sqlite]:
```suggestion
/* Copyright 2020 The Matrix.org Foundation C.I.C.
```

**Comment by richvdh** [comment]
> I trust this is what we want to do

I think I managed to persuade @erikjohnston that it is what we want to do.

**Comment by erikjohnston** [comment]
yeaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaah

**Comment by erikjohnston** [comment]
what's the worst that can happen

**Pull request merged**